pkg/api/v1/types.go

```diff
@@ -2048,7 +2048,7 @@ type NodeSpec struct {
 // DaemonEndpoint contains information about a single Daemon endpoint.
 type DaemonEndpoint struct {
 	// Port number of the given endpoint.
-	Port int32 `json:port`
+	Port int32 `json:"port"`
```

Comment:
Looks like it is used, here:
https://github.com/kubernetes/kubernetes/blob/d2d349bc84dd3a0aa8b220a8cbbbb8b240dc516d/pkg/registry/node/strategy.go#L183

So, we can't fix this, unfortunately.